### PR TITLE
Bug 1756132: Remove incorrect optional tag from new ingress field

### PR DESCRIPTION
**Pull request by ironcladlou**
`loadBalancer` is a new field that contains `scope`, and `loadBalancer` is
optional (correct). However, `loadBalancer.scope` _should_ be required. This
commit fixes that.

### CODE REVIEW

File: operator/v1/types_ingress.go [hunk 1]
```diff
@@ -188,7 +188,6 @@ type LoadBalancerStrategy struct {
 	// scope indicates the scope at which the load balancer is exposed.
 	// Possible values are "External" and "Internal".  The default is
 	// "External".
-	// +optional
```
Comment (Miciah) [inline operator/v1/types_ingress.go]: Looks reasonable to me, but we need clarification from @deads2k regarding https://github.com/openshift/api/pull/312/files#r294390499.

Comment (ironcladlou) [reply operator/v1/types_ingress.go]: Since the package default is Optional, does this need to be explicitly required?

Comment (ironcladlou) [reply operator/v1/types_ingress.go]: I marked this explicitly required and moved the defaulting docs up to the actual point of defaulting (the optional `loadBalancer` field).

**Pull request merged**